src/components/screens/Validacion/components/index.module.scss

```
@@ -0,0 +1,30 @@
+.pantallaValidacion{
+    width: 1024px;
+    margin: 0 auto;
+}
```

Cuidado con la identación en todo el archivo (se están usando 4 espacios y en el proyecto usan 2):
```suggestion
.pantallaValidacion {
  width: 1024px;
  margin: 0 auto;
}
```